src/main/java/com/google/api/codegen/transformer/SampleTransformer.java

```diff
@@ -251,6 +255,18 @@ private MethodSampleView generateSample(
                     initCodeContext.cloneWithEmptySymbolTable())); // to avoid symbol collision
     ImportSectionView sampleImportSectionView =
         sampleImportTransformer().generateImportSection(methodContext);
+    SampleFunctionDocView sampleFunctionDocView =
+        SampleFunctionDocView.newBuilder()
+            .paramDocLines(paramDocLines(methodContext, initCodeView))
+            .mainDocLines(
+                ImmutableList.<String>builder()
+                    .addAll(
+                        methodContext
+                            .getNamer()
+                            .getDocLines(valueSet.getDescription(), DOC_LINE_MAX_WIDTH, false))
```

Comment:
@vchudnov-g Actually sorry, we do need the boolean flag `reformat` here. The `false` here should have been `true`, i.e.,  we need to reformat the function description here.

Reply:
OK, then in the test cases, please add special characters to the appropriate sample config to ensure they get escaped correctly (ie once).  eg `// do foo & bar if a < 5 `

Reply:
Sure!